brainmap-cms/static/brainmap-cms/css/src/site.css

```diff
@@ -0,0 +1,36 @@
+/* Organize via ITCSS */
+/* SEE: https://confluence.tacc.utexas.edu/x/IAA9Cw */
+
+
+
+
+
+/* SETTINGS */
+
+/* https://github.com/TACC/Core-Styles/blob/22783d8/src/lib/_imports/settings/color.css */
+:root {
+  /* Accent */
+  --global-color-accent--x-light: hsl(229 45% 77%);
+  --global-color-accent--light: hsl(229 45% 60%);
+  --global-color-accent--normal: hsl(229 45% 44%); /* from logo */
+  --global-color-accent--dark: hsl(229 45% 35%);
+  --global-color-accent--x-dark: hsl(229 45% 30%);
+
+  --global-color-accent--alt: hsl(229 22% 77%);
+  --global-color-accent--weak: hsl(229, 45%, 52%, 0.25);
+
+  /* Link */
+  --global-color-link-on-light--normal: var(--global-color-accent--normal);
+  --global-color-link-on-dark--normal: var(--global-color-accent--x-light);
```

Comment:
<details><summary>The accent color is whatever from Core, until I get a logo or other brand identification to work with.</summary>

1. I began with the dull cobalt blue from the logo.
2. I modified the colors to match differences in Core.
3. I [verified accessible contrast of `--global-color-accent--x-light` atop `--global-color-primary--xx-dark`](https://webaim.org/resources/contrastchecker/?fcolor=AAB4DF&bcolor=222222).*

<sup>* The [`--global-color-primary--xx-dark` is bkgd. color of dark sections](https://github.com/TACC/Core-Styles/blob/7dbe24e/src/lib/_imports/objects/o-section.css#L111). The `--global-color-accent--x-light` is used for links on dark sections.</sup>

</details>

<details><summary>Screenshots</summary>

| core-styles | brainmap-cms |
| - | - |
| ![core-styles](https://user-images.githubusercontent.com/62723358/202316861-e05af0f7-f63f-47fd-b1fd-f2015d88b2ca.png) | ![brainmap-cms](https://user-images.githubusercontent.com/62723358/202316862-4dcc6a0e-92cf-497e-8d7d-eb9e956ce21e.png) |

</details>